.gitignore

```
@@ -1,3 +1,4 @@
 target/
 .idea
-portfolio.iml
\ No newline at end of file
+portfolio.iml
+walkthroughs/
```

```suggestion
walkthroughs/

```
I think git likes to have newlines at the end of file.